### DISCUSSION TITLE
OSS projects or public repositories using Dagster

### DISCUSSION THREAD

**Discussion by yamhirotoCS** [dagster-io/dagster#18147] [General]
Hello,
Does anyone of you know any OSS projects or publicly accessible repositories using Dagster at scale? 

Our team is past the 'tutorial' code and to improve our project, we do a mix of: reading once again the documentation, browsing github discussion and the official slack. We would also love to see how other team organised theirs as well.

I saw this thread, but it's limited to where it's being used. 
https://github.com/dagster-io/dagster/discussions/7218

Perhaps the Dagster team itself uses their own tool?

thanks !

**Comment by dduong1603** [comment]
Dagster team opens up their implementation [here](https://github.com/dagster-io/dagster-open-platform) but they are still migrating things over from their internal place so it might take a while 😄
